integration/kubernetes/helm-chart/alluxio/values.yaml

```diff
@@ -46,6 +46,11 @@ properties:
 # Use labels to run Alluxio on a subset of the K8s nodes
 # nodeSelector: {}
 
+# A list of k8s Node taints to allow scheduling on.
+# See the Kubernetes docs for more info:
+# - https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/
+# tolerations: []
```

Comment:
Does it make sense to add an example in the comment?

Reply:
yes, as many make a mistake between `list` and `maps` while passing values here.

Reply:
Ah okay, I'll specify an example